### ISSUE TITLE
Annotation overspills in UI  + recognise hyperlinks

### ISSUE THREAD

**Issue by mcourtot** [EBISPOT/OLS#465]
https://www.ebi.ac.uk/ols/ontologies/duo and attached screenshot
![Screenshot 2021-02-25 at 08 37 55](https://user-images.githubusercontent.com/2214135/109126099-cca85f00-7744-11eb-8236-b9d8cb179a2a.png)

Bug: the links towards the GH contributors list overspills on the right of the ontology information box.
Enhancement: it would be great if the link could also be recognised and rendered as an hyperlink.

**Comment by mcourtot** [comment]
Same is true at https://www.ebi.ac.uk/ols/ontologies/duo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FDUO_0000007&viewMode=PreferredRoots&siblings=false for the "Term information" box

![Screenshot 2021-02-25 at 08 42 50](https://user-images.githubusercontent.com/2214135/109126727-7f78bd00-7745-11eb-9cca-a2dd2d90ac66.png)

**Comment by henrietteharmse** [comment]
The text that is overspilling is also an enhancement. The text areas are good at wrapping at the end of words. To try and resolve this, all text will need to be analysed to figure out whether there is by chance a long string that needs to be broken up artificially. This has performance implications. Hence, this is not something we can implement. Similarly figuring out whether arbitrary text contains hyperlinks will have performance implications. 

For both these requests I will prefer having dedicated annotations that the community can agree on: 

- See Also: Which can only contain a list of hyperlinks
- Contributors Links: Which can only contain a list of hyperlinks

**Comment by henrietteharmse** [comment]
Ah! Just had a chat with @zoependlington. Apparently [http://www.w3.org/2000/01/rdf-schema#seeAlso](http://www.w3.org/2000/01/rdf-schema#seeAlso) is already used in EFO and Mondo.

**Comment by matentzn** [comment]
This is great. Now this would be amazing, if it could be done in the same wash: hasDbXref. This points to a curie sometimes (most often), which ols knows how to turn into an iri. [Example](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0021060&viewMode=PreferredRoots&siblings=false):

![image](https://user-images.githubusercontent.com/7070631/109137114-c3bd8a80-7750-11eb-88e4-043f014b3408.png)

Not necessary for this ticket, but would be awesome: could we make the orphanet and efo links clickable as well?
Orphanet:536391
EFO:1001502

**Comment by henrietteharmse** [comment]
@matentzn Implementation wise that is a different issue because EFO:1001502 is not actually a link but needs to be looked up to find the link and then rendered as link. The problem is to do this lookup efficiently. Me and @udp have discussed this extensively last year without finding an easy way to add this into OLS.

**Comment by matentzn** [comment]
I thought The OLS api can do this! But yes different issue. Just wanted to chime in!

**Comment by henrietteharmse** [comment]
Yes, OLS indeed has an API call that can look up a single term. However, this is not efficient for use in this case as it will result in a sharp increase in the amount of traffic generated for each ontology term visited for each user. What we need here a cache with latency close to 0. Otherwise this will result in OLS pages load much slower.
